### PR TITLE
Add Ping method.

### PR DESCRIPTION
**Pull request by andreiavrammsd**
Checks if instances are alive.

```
errorCollector := make(chan error)
m := memcache.New("/var/run/memcached.sock")

go func() {
	tick := time.NewTicker(time.Second)
	for range tick.C {
		if err := m.Ping(); err != nil {
			errorCollector <- err
		}
	}
}()

go func() {
	for err := range errorCollector {
		fmt.Println(err)
	}
}()
```

### CODE REVIEW

**Comment by andreiavrammsd** [comment]
I signed it!

File: memcache/memcache.go [hunk 1]
```diff
@@ -644,6 +669,11 @@ func (c *Client) DeleteAll() error {
 	})
 }
 
+// Ping checks if instances are alive.
```
Comment (bradfitz) [inline memcache/memcache.go]: This doesn't make it clear whether it checks requires any or all servers to be up to return a nil error.

File: memcache/memcache.go [hunk 2]
```diff
@@ -402,24 +402,10 @@ func (c *Client) flushAllFromAddr(addr net.Addr) error {
 // ping sends the version command to the given addr
 func (c *Client) ping(addr net.Addr) error {
 	return c.withAddrRw(addr, func(rw *bufio.ReadWriter) error {
-		if _, err := fmt.Fprintf(rw, "version\r\n"); err != nil {
+		if _, err := fmt.Fprintf(rw, "info clients\r\n"); err != nil {
```
Comment (bradfitz) [inline memcache/memcache.go]: Why did you add this commit? I didn't ask you to change this during review.

What was wrong with version? It actually worked. What is "info clients"? I see no reference to that in the protocol or code.

Comment (andreiavrammsd) [reply memcache/memcache.go]: I tested again the ping command and I got:
`memcache: unexpected response line from ping: "-ERR unknown command `version`, with args beginning with: \r\n"`

Then I switched to [info](https://redis.io/commands/info) and tested on redis 5 and 4.

Comment (andreiavrammsd) [reply memcache/memcache.go]: Ah, sorry, that's what you get when don't sleep enough... I'm reverting it now.
